src/Doctrine/EntityRegenerator.php

```diff
@@ -129,8 +129,7 @@ public function regenerateEntities(string $classOrNamespace): void
                             ->setIsNullable($getIsNullable($mapping))
                             ->setTargetClassName($mapping['targetEntity'])
                             ->setTargetPropertyName($mapping['inversedBy'])
-                            ->setMapInverseRelation(null !== $mapping['inversedBy'])
-                        ;
+                            ->setMapInverseRelation(null !== $mapping['inversedBy']);
```

Comment:
revert.

Please do not change things that is not related to the PR.

Reply:
done